src/Assignment-3/assignment3.css

```diff
@@ -5,3 +5,80 @@
 width: 100%;
 height: 100%;
 }
+*{
+    margin:0;
+    padding:0;
+}
+body{
+    background-color: #f2f2f2;
+}
+.first{
+    margin-top:5vw;
+    margin-left: 5vw;
+    margin-right:5vw;
+    width:90vw;
+    
+    display:flex;
+    flex-wrap: wrap-reverse;
+    flex-direction: row;
+    height:75vh;
+}
+.content-first{
+    display:flex;
+    width:50%;
+    flex-direction: column;
+    justify-content:space-evenly;
+    
+}
+#render-wiki{
+    display:inline-block;
+    height:100%;
+    width:100%;
+}
+.iframe-parent{
+    width: 50%;
+}
+h1{
+    font-size: 3rem;
+    transform: translate();
+}
+button{
+    color:white;
+    border-radius: 0;
+    border: None;
+    height:10%;
+    width: 20%;
+    background-color: black;
+}
+button:hover{
+    background-color: white;
+    color:black;
+    border: 2px solid black;
+}
+.content-second{
+    padding-top:13vh;
+    height:50vh;
+    display:flex;
+   
+    flex-direction: column;
+    justify-content: space-evenly;
+    
+}
+.h12{
+    text-align: center;
+}
+.p2{
+    width:30vw;
+    text-align:center ;
+    align-self: center;
```

Comment:
why both text-align and align-self???

Reply:
one is aligning text justification and one is alignining the text p alignment.. @chitha-srinath